target/linux/ath79/dts/qca9558_domywifi_dw33d.dts

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	compatible = "domywifi,dw33d", "qca,qca9558";
+	model = "DomyWifi DW33D";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: system {
+			label = "dw33d:blue:system";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		mmc {
+			label = "dw33d:blue:mmc";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port0>;
+			linux,default-trigger = "usbport";
+		};
+
+		usb1 {
+			label = "dw33d:blue:usb1";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port1>;
+			linux,default-trigger = "usbport";
+		};
+
+		wlan-2g {
```

Comment:
I would personally use "wlan2g", but that's a matter of taste. The label can't be changed, as it's determined by ar71xx.